### DISCUSSION TITLE
How to include extra protoc plugins in Quarkus using Gradle?

### DISCUSSION THREAD

**Discussion by ranjanashish** [quarkusio/quarkus#27278] [Q&A]
I am implementing a gRPC server using Quarkus gRPC extension with Gradle as my build tool. How can I instruct Quarkus to include another protoc plugin for code generation. The plugin that I am trying to include is https://github.com/envoyproxy/protoc-gen-validate#java.

[a comment by an automated account is not shown]

**Reply by ranjanashish** [reply]
Any help on this will be great.

If this support does not exist today, I will be happy to contribute. Since I am new to Quarkus, so a few pointers as to where to start will be helpful.

**Reply by glefloch** [reply]
Hi @ranjanashish, this is not possible for now. we use the `protobuf-java` library without plugin. I will have a look at how we can include this kind of dependency in the generation process.

**Reply by ranjanashish** [reply]
Thank you so much for confirming. I will a create an issue for "Feature request / Enhancement".

**Reply by ranjanashish** [reply]
Created issue - https://github.com/quarkusio/quarkus/issues/27388

**Reply by venkatesh-i2i** [reply]
@glefloch, is there any way to disable the Quarkus Proto generation? so that i can use my other plugin by configuring protoc compile in gradle
